packages/flutter/test/widgets/binding_live_test.dart

```diff
@@ -0,0 +1,55 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/material.dart';
+import 'package:flutter_test/flutter_test.dart';
+
+// This file is for tests for WidgetsBinding that require a `LiveTestWidgetsFlutterBinding`
+void main() {
+  LiveTestWidgetsFlutterBinding();
+  testWidgets('ReportTiming callback records the sendFramesToEngine when it was scheduled', (WidgetTester tester) async {
+    // This test needs LiveTestWidgetsFlutterBinding for multiple reasons.
+    // First, this was the environment that this bug was discovered.
+    // Second, LiveTestWidgetsFlutterBinding doesn't override
+    // scheduleWarmUpFrame, so that the test can start with no frames rendered
+    // allowing deferFirstFrame to work (see below).
+
+    // At the beginning of a `testWidgets`, `TestWidgetsFlutterBinding` pumps a
+    // fixed dummy widget using `runApp`, then `resetFirstFrameSent`. Although
+    // the dummy widget is rendered, its reportTiming callback from the engine
+    // has not arrived yet.
+    //
+    // Note that the `runApp` call also schedules a warm up frame, which would
+    // have directly rendered the frame if the test had used
+    // `AutomatedTestWidgetsFlutterBinding`.
```

Comment:
This doesn't seem super relevant and adds more confusion to what is actually going on here. Can we just drop this? Also note that "note that" is banned: https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#avoid-empty-prose